app/src/main/res/layout/activity_about_us.xml

```diff
@@ -1,57 +1,413 @@
+                    android:text="@string/about_us_donation_from_india_bank_name_value" />
+            </TableRow>
+
+
+            <TableRow
+                android:layout_width="fill_parent"
+                android:layout_height="0dp"
+                android:layout_margin="1dp"
+                android:layout_weight="1"
+                android:paddingStart="@dimen/view_padding"
+                android:paddingEnd="@dimen/view_padding">
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_column="0"
+                    android:layout_margin="1dp"
+                    android:layout_weight="0.5"
+                    android:text="@string/about_us_donation_ac_no" />
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_column="1"
+                    android:layout_margin="1dp"
+                    android:layout_weight="0.5"
+                    android:text="@string/about_us_donation_from_india_ac_no_value" />
+            </TableRow>
+
+            <TableRow
+                android:layout_width="fill_parent"
+                android:layout_height="0dp"
+                android:layout_margin="1dp"
+                android:layout_weight="1"
+                android:paddingStart="@dimen/view_padding"
+                android:paddingEnd="@dimen/view_padding">
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_column="0"
+                    android:layout_margin="1dp"
+                    android:layout_weight="0.5"
+                    android:text="@string/about_us_donation_ifsc" />
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_column="1"
+                    android:layout_margin="1dp"
+                    android:layout_weight="0.5"
+                    android:text="@string/about_us_donation_from_india_ifsc_value" />
+            </TableRow>
+        </TableLayout>
+
+        <TextView
+            android:id="@+id/textview_about_us_international_donation_heading"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="@dimen/view_padding"
+            android:text="@string/about_us_donation_international_heading"
+            android:textSize="@dimen/text_size_small"
+            android:textStyle="bold" />
+
+        <TableLayout
+            android:id="@+id/donation_info_international"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:stretchColumns="0,1">
+
+            <TableRow
```

Comment:
This TableRow looks same, can we create another layout file and include it here and then set the text?